### DISCUSSION TITLE
Update_topics with another vectorizer model and representation model

### DISCUSSION THREAD

**Discussion by Nee-DS** [MaartenGr/BERTopic#1255] [Q&A] [answered]
I have already used representation_model = KeyBERTInspired() and vectorizer = CountVectorizer(stop_words="english") in the Bertopic Model pipeline. 
I am updating the topics by using 

vectorizer_model2= CountVectorizer(tokenizer=LemmaTokenizer()) 
representation_model2 = MaximalMarginalRelevance(diversity=0.7)

new_topics=topic_model.update_topics(text,topics, top_n_words=10, vectorizer_model=vectorizer_model2,representation_model=representation_model2) 

IS IT CORRECT?! Theoretically?

**Comment by MaartenGr** [comment, marked as answer]
Yes, the `.update_topics` is meant for this kind of purpose. It allows you to easily change the topic representations in order to fine-tune your model.

**Reply by sgautam666** [reply]
Hello Maarten, I am trying to use update_topics() method. In the example above, can we use the 'updated_model' something like updated_model.get_topic_info() ? 
```
updated_model = topic_model.update_topics(text, n_gram_range(2,3)) 
updated_model.get_topic_info() 
```
When I try this apporach, I get 'NonType' object has no attribute 'get_topic_info'.
Thanks

**Reply by MaartenGr** [reply]
It would be best to follow along with the example in the [documentation](https://maartengr.github.io/BERTopic/getting_started/topicrepresentation/topicrepresentation.html#update-topic-representation-after-training) to make sure that you ran everything correctly. I should note that it updates the original model and does not return a new model, so this does not work.

**Reply by sgautam666** [reply]
Thank you!
